__tests__/app.test.js

```diff
@@ -233,73 +233,73 @@ describe("Patch /api/reviews/:review_id", () => {
   });
 });
 
-
 describe("GET /api/reviews/:review_id/comments", () => {
   test("returns an array of objects and a 200 status", () => {
     return request(app)
       .get("/api/reviews/2/comments")
       .expect(200)
-      .then((res)=>{
-        expect(res.body.comments).toBeInstanceOf(Array)
-        res.body.comments.forEach((comment)=>{
-          expect(comment).toBeInstanceOf(Object)
-        })
-      })
+      .then((res) => {
+        expect(res.body.comments).toBeInstanceOf(Array);
+        res.body.comments.forEach((comment) => {
+          expect(comment).toBeInstanceOf(Object);
+        });
+      });
   });
```

Comment:
This whole test is redundant as the test after the next  checks the same thing, plus more.